examples/sample_runtime/src/main.rs

```diff
@@ -301,6 +298,7 @@ fn main() {
 
     let blockchain_ref = blockchain.as_ref().to_owned();
     let node = Node::with_blockchain(blockchain, channel, node_cfg, None);
+    let shutdown_handle = node.shutdown_handle();
```

Comment:
`ShutdownHandle` is now separated from `ApiSender`, although it uses `ApiSender` internally. This is to provide separation of concerns and prevent misuse.